content/en/monitors/create/types/apm.md

```diff
@@ -94,11 +94,15 @@ For detailed instructions on the advanced alert options (no data, evaluation del
 
 ### Select alert conditions
 
-* Trigger when the metric is `above` or `above or equal to`
-* the threshold during the last `5 minutes`, `15 minutes`, `1 hour`, etc. or `custom` to set a value between 5 minutes and 48 hours.
+* Trigger when the query result is `above`, `above or equal to`, `below`, or `below or equal to`.
+* The threshold during the last `5 minutes`, `15 minutes`, `1 hour`, or `custom` to set a value between 5 minutes and 48 hours.
 * Alert threshold: `<NUMBER>`
 * Warning threshold: `<NUMBER>`
 
+#### No data and below alerts
```

Comment:
1. Why do you think we should add this extra section? Don't you think it's expressed by the above section already?
2. If we decide to keep it, we might want to update it a little so that it is not only geared at multi alert monitors - WDYT?